src/vs/base/parts/tree/browser/treeImpl.ts

```diff
@@ -205,6 +205,13 @@ export class Tree implements _.ITree {
 	getContentHeight(): number {
 		return this.view.getContentHeight();
 	}
+
+	/**
+	 * Sets the `aria-label` attribute of the underlying root HTML element.
+	 */
+	public set ariaLabel(value: string) {
```

Comment:
Let's avoid modifying vs code - just call getHTMLElement in the sql code and call setAttribute on that directly

Reply:
Oh this is already in a sql block...eh yeah ok this is fine.

(VS deleted this tree control a long time back anyways so it doesn't matter much at this point anyways)